<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Running</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/application.qrc">
    <normaloff>:/icon</normaloff>:/icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="calendarPage">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="CalendarWidget" name="calendarWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="eventPage">
       <layout class="QVBoxLayout" name="eventPageVerticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="statisticsPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Search type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="statisticsPageEventsPushButton">
            <property name="text">
             <string>Events</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="statisticsPageShoesPushButton">
            <property name="text">
             <string>Shoes</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>96</width>
             <height>26</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="StatisticsWidget" name="statisticsWidget" native="true">
              <property name="palette">
               <palette>
                <active>
                 <colorrole role="Base">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Window">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="Base">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Window">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="Base">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Window">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="actionEdit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionCalendar"/>
    <addaction name="actionStatistics"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="actionSystemInformations"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCalendar"/>
   <addaction name="actionStatistics"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="actionEdit"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../../resources/application.qrc">
     <normaloff>:/images/event_add</normaloff>:/images/event_add</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../../resources/application.qrc">
     <normaloff>:/images/event_delete</normaloff>:/images/event_delete</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../resources/application.qrc">
     <normaloff>:/images/exit</normaloff>:/images/exit</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../resources/application.qrc">
     <normaloff>:/images/about</normaloff>:/images/about</iconset>
   </property>
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../../resources/application.qrc">
     <normaloff>:/images/event_edit</normaloff>:/images/event_edit</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionCalendar">
   <property name="icon">
    <iconset resource="../../resources/application.qrc">
     <normaloff>:/images/calendar</normaloff>:/images/calendar</iconset>
   </property>
   <property name="text">
    <string>Calendar</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../resources/application.qrc">
     <normaloff>:/images/settings</normaloff>:/images/settings</iconset>
   </property>
   <property name="text">
    <string>Settings ...</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="icon">
    <iconset resource="../../resources/application.qrc">
     <normaloff>:/images/statistics</normaloff>:/images/statistics</iconset>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionSystemInformations">
   <property name="icon">
    <iconset resource="../../resources/application.qrc">
     <normaloff>:/images/information</normaloff>:/images/information</iconset>
   </property>
   <property name="text">
    <string>System Informations ...</string>
   </property>
   <property name="toolTip">
    <string>System Informations</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header>widgets/calendarwidget/calendarwidget.h</header>
  </customwidget>
  <customwidget>
   <class>StatisticsWidget</class>
   <extends>QWidget</extends>
   <header>widgets/statisticswidget/statisticswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>calendarWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/application.qrc"/>
 </resources>
 <connections/>
</ui>
